organizational/slack.rst

```diff
@@ -1,12 +1,12 @@
 HacSoc Slack
 ============
 
-`Hacsoc Slack`_ 
+`Click here to go to the official HacSoc Slack.`_ 
 
 .. _Hacsoc Slack: https://hacsoc.slack.com
```

Comment:
This change will almost certainly break the link. Notice how the reference on line 6 matches the link text on line 4. You can either make them match again, or switch to the inline link syntax:

    `link text <link target>`_

[Quick reference](http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets). Be sure to check these things with a `make html` - there will be "compiler warnings" for broken links, etc.